pkg/products/rhssouser/prometheusRules.go

```diff
@@ -72,6 +72,77 @@ func (r *Reconciler) newAlertsReconciler(logger l.Logger, installType string) re
 					},
 				},
 			},
+
+			//SLO-availability-alerts:
+			//https://sre.google/workbook/alerting-on-slos/
+			//https://promtools.dev/alerts/errors
+			{
+				AlertName: "rhoam-user-sso-slo-availability-alerts",
+				Namespace: r.Config.GetOperatorNamespace(),
+				GroupName: "rhoam-user-sso-slo-availability.rules",
+				Rules: []monitoringv1.Rule{
+					{
+						Alert: "RHOAMUserSsoAvailability5mto1hErrorBudgetBurn",
+						Annotations: map[string]string{
+							"sop_url": resources.SopUrlRHOAMUserSsoAvailabilityAlert,
+							"message": "High 5m and 1h error budget burn for RHOAM SSO User",
+						},
+						Expr: intstr.FromString(`
+							sum( sum(rate(haproxy_backend_http_responses_total{route=~"^keycloak.*", exported_namespace="redhat-rhoam-user-sso", code="5xx"}[5m]))
```

Comment:
Out of curiousity, are these alerts for RHMI also ? :thinking: 

If they are, then currently I dont think they would work for an RHMI install with the hardcoded `redhat-rhoam` for namespace references so might want to use `r.Config.GetNamespace()` call instead like :thinking: 
* https://github.com/integr8ly/integreatly-operator/blob/master/pkg/products/rhsso/prometheusRules.go#L30

Reply:
Hi Kevin, thank you very much for your Review and comments.
New Alerts updated now to support RHMI.